faker/tests/requirements.txt

```diff
@@ -0,0 +1 @@
+mock==1.0.1
```

Comment:
Great. I updated the `*.yml`s (I needed to add one for CircleCI) and updated the documentation in the README for running tests.

Any other changes you'd like me to make while waiting for tests to finish?